tests/fuzz/fuzz_h2_decoder_correct.c

```diff
@@ -270,8 +270,22 @@ int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
                 if (settings_count > 0) {
                     settings_array = aws_mem_calloc(allocator, settings_count, sizeof(struct aws_h2_frame_setting));
                     for (size_t i = 0; i < settings_count; ++i) {
-                        aws_byte_cursor_read_be16(&input, &settings_array[i].id);
-                        aws_byte_cursor_read_be32(&input, &settings_array[i].value);
+                        uint16_t id;
+                        uint32_t value;
+                        aws_byte_cursor_read_be16(&input, &id);
+                        aws_byte_cursor_read_be32(&input, &value);
```

Comment:
initialize `id` and `value` to 0
that way if input runs out of data and doesn't bother setting the out-param, we'll still get predictable results